src/Networks/Blockcore.Networks.HomeCoin/Blockcore.Networks.HomeCoin.csproj

```diff
@@ -0,0 +1,37 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Authors>Blockcore</Authors>
+    <AssemblyTitle>Blockcore.Networks.HomeCoin</AssemblyTitle>
+    <AssemblyName>Blockcore.Networks.HomeCoin</AssemblyName>
+    <PackageId>Blockcore.Networks.HomeCoin</PackageId>
+    <GeneratePackageOnBuild>False</GeneratePackageOnBuild>
+    <IsPackable>true</IsPackable>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <LangVersion>latest</LangVersion>
+    <Description>Home Crypto Coin</Description>
+    <PackageProjectUrl>https://homecoin.ru</PackageProjectUrl>
```

Comment:
I really like this idea about overriding the default Blockcore properties. All individual network definitions should do the same.